### ISSUE TITLE
Question about Readme

### ISSUE THREAD

**Issue by Jimmy-Hu** [respeaker/usb_4_mic_array#14]
Hello,

I read the README. In the description about the "Realtime sound source localization and tracking", the third step said "upgrade your usb 4 mic array with i6_firmware.bin which includes 4 channels raw audio data." I wonder that the file "i6_firmware.bin" is referring to the file "6_channels_firmware.bin" in the master branch ? I can't download the file "i6_firmware.bin" by the URL on README.

**Comment by Jimmy-Hu** [comment]
Thank you so much. I try to use the file "6_channels_firmware.bin" to upgrade the firmware.

**Comment by KillinTimePrisonBlues45** [comment]
Thank you so much for leaving a slimy ass trail cuz I've got it all now so I hope u enjoy jail. My privacy has been violated my phone services stolen 9 phones broken from over heating or being thrown. Oh how I wish it said ur actual name not a login credential cuz I would drive straight to ur location and give u a free no money back no obligation actually id give u what u have given me no choice no rights no way to protect yourself just suck it up bitch while I beat ur helpless punk fucking ass. Oh believe me after being recorded my pics taken my privacy rights more than violated I didn't sign anything I didn't consent so before u change meeting me I'd suggest u repent. Im getting a lawyer & im coming after u & every person involved I've got diabetes so bad they took me off work to reduce stress while my husband & a whore chose to search on pinterest for foods that contain sulfa to which im deathly allergic. Ive got photos that will turn ur stomach of a beating & choking nearly to death i took yet u all smugly sit & use technology to fuck with my life without any regard to my health or safety may u each get the max cuz thats the least u deserve. I didn't put this shit on my fb my phone or brand new laptop so I suggest u all fix it take it completely off my shit cuz my hubby is selling all my pics cuz they go straight to his skype & Microsoft accounts azure & cloud to. Sad to think someone could hate me that much or make it look like my husband did it I no longer care who did if it all goes away if it doesn't im on the verge of a psychotic snap & my hubby's in prison till 2027. Oh how I love him he thinks I'm cute crazy & dumb but it took me less than a year to figure it all out how long u geniuses been trying to set the shit up? Over 2 years & finally I can breathe and relax cuz if it don't get taken down
